src/config/index.js

```diff
@@ -0,0 +1,84 @@
+import defaultConfig from './default'
```

Comment:
Could you modify `@kadira/storybook`'s babel-config script to be modular and be called from the outside.

Then we can simply import it here. Duplicating the code may be hard to manage in the future.